adapters/gumgum/gumgum.go

```diff
@@ -176,6 +176,15 @@ func preprocess(imp *openrtb2.Imp) (*openrtb_ext.ExtImpGumGum, error) {
 		}
 	}
 
+	if gumgumExt.Product != "" {
+		var err error
+		//imp.Ext, err = json.Marshal(`{ "product": "%s" }`, gumgumExt.Product)
+		imp.Ext, err = json.Marshal(map[string]string{"product": gumgumExt.Product})
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
should update json tests for above change

Reply:
Added a couple of valid/invalid params for test cases. Thanks